### DISCUSSION TITLE
Ubuntu GUI stock at loading wallet

### DISCUSSION THREAD

**Discussion by Tongie969** [Chia-Network/chia-blockchain#2392] [General]
Hi there! 

I am so frustrated trying to set up chia-blockchain on Ubuntu. GUI is loading, but stocks at loading wallet. I did not create one for now. Browser did not open a window for setting one up. 

Here is the console output, with error. 

(venv) $ npm run electron &
(venv) $ 
> chia-blockchain@ electron /Chia/chia-blockchain/chia-blockchain-gui
> electron .

/Chia/chia-blockchain/app.asar.unpacked/daemon
false
Error loading config - using defaults
startUrl file:///Chia/chia-blockchain/chia-blockchain-gui/build/renderer/index.html
/Chia/chia-blockchain/app.asar.unpacked/daemon
false
/Chia/chia-blockchain/app.asar.unpacked/daemon
false
Running python script
Script ../chia/daemon/server.py
child process success
(node:51957) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
No cert
Have cert
stderr: Traceback (most recent call last):
  File "../chia/daemon/server.py", line 962, in <module>
    main()
  File "../chia/daemon/server.py", line 958, in main
    return run_daemon(DEFAULT_ROOT_PATH)
  File "../chia/daemon/server.py", line 952, in run_daemon
    return asyncio.get_event_loop().run_until_complete(async_run_daemon(root_path))
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "../chia/daemon/server.py", line 944, in async_run_daemon
    await ws_server.start()
  File "../chia/daemon/server.py", line 138, in start
    self.websocket_server = await serve(
  File "/Chia/chia-blockchain/venv/lib/python3.8/site-packages/websockets/server.py", line 965, in __await_impl__
    server = await self._create_server()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1463, in create_server
    raise OSError(err.errno, 'error while attempting '
OSError: [Errno 99] error while attempting to bind on address ('::1', 55400, 0, 0): cannot assign requested address
closing code: 1

Thanks in Advance! 

Tongie

**Comment by Tongie969** [comment]
![image](https://user-images.githubusercontent.com/63996344/115737148-29627780-a38c-11eb-8c5c-590ce008ec70.png)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
